src/components/Sidebar/NavContent.tsx

```diff
@@ -82,83 +83,34 @@ const NavContent: React.FC<NavContentProps> = ({ handleDrawerToggle }) => {
-                                  className={"bond"}
-                                  onClick={handleDrawerToggle}
-                                >
-                                  <Typography variant="body2">
-                                    {bond.displayName}
-                                    <span className="bond-pair-roi">
-                                      <DisplayBondDiscount key={bond.index} bond={bond} />
-                                    </span>
-                                  </Typography>
-                                </Link>
-                              );
-                            })}
-                          </AccordionDetails>
-                        </Accordion>
-                      )}
-                      {sortedInverseBonds.length > 0 && (
-                        <Accordion className="discounts-accordion" square defaultExpanded={true}>
-                          <AccordionSummary
-                            expandIcon={
-                              <ExpandMore className="discounts-expand" style={{ width: "18px", height: "18px" }} />
-                            }
-                          >
-                            <Typography variant="body2">
-                              <Trans>Inverse Bonds</Trans>
-                            </Typography>
-                          </AccordionSummary>
-                          <AccordionDetails>
-                            {sortedInverseBonds.map((bond, i) => {
-                              return (
-                                <Link
-                                  component={NavLink}
-                                  to={`/bonds/inverse/${bond.index}`}
-                                  key={i}
-                                  className={"bond"}
-                                  onClick={handleDrawerToggle}
-                                >
-                                  <Typography variant="body2">
-                                    {bond.displayName}
-                                    <span className="bond-pair-roi">
-                                      <DisplayBondDiscount key={bond.index} bond={bond} />
-                                    </span>
-                                  </Typography>
-                                </Link>
-                              );
-                            })}
-                          </AccordionDetails>
-                        </Accordion>
-                      )}
-                    </div>
-                  </div>
+                  <NavItem to="/bonds" icon="bond" label={t`Bond`}>
+                    {sortedBonds.map((bond, i) => (
+                      <NavItem
+                        to={`/bonds/${bond.index}`}
+                        key={i}
+                        onClick={handleDrawerToggle}
+                        label={bond.displayName}
+                        chip={`${bond.discount && trim(bond.discount * 100, 2)}%`}
```

Comment:
Can this not be a chip? Chips should be used for alerts or filter labels. This is a nit but would rather discount text color be red/green text and no chip.